packages/ui-kit/src/Toggler/Toggler.fixture.tsx

```diff
@@ -16,8 +17,14 @@ export default {
       foo: false,
       bar: true,
       baz: false,
+      payment: false,
```

Comment:
Since the example for two-directional toggle is a bit more complex, let's give this a more informative name, like "isAnnual"